src/main/resources/assets/gregtech/lang/en_us.lang

```diff
@@ -3524,6 +3526,7 @@ gregtech.advancement.ultimate_voltage.80_diamericium_titanium_coil.desc=Craft a
 
 
 behaviour.softhammer=Activates and Deactivates Machines
+behaviour.hammer=Turns on and off Muffling on Machines (by smashing Them)
```

Comment:
I'd suggest changing `smashing` to `hitting`. Sounds better in my head personally. Also, `Them` being capital looks weird in this case, would suggest making it lowercase.